rules/build_sector.smk

```diff
@@ -850,13 +858,13 @@ rule build_biomass_potentials:
     params:
         biomass=config_provider("biomass"),
     input:
-        enspreso_biomass="data/ENSPRESO_BIOMASS.xlsx",
-        eurostat="data/eurostat/Balances-April2023",
-        nuts2="data/nuts/NUTS_RG_03M_2013_4326_LEVL_2.geojson",
+        enspreso_biomass=rules.retrieve_enspreso_biomass.output["xlsx"],
+        eurostat=rules.retrieve_eurostat_balances.output["directory"],
+        nuts2=rules.retrieve_eu_nuts_2013.output["shapes_level_2"],
         regions_onshore=resources("regions_onshore_base_s_{clusters}.geojson"),
-        nuts3_population=ancient("data/bundle/nama_10r_3popgdp.tsv.gz"),
+        nuts3_population=ancient(rules.retrieve_nuts3_population.output["gz"]),
```

Comment:
@coroa @fneum Do you remember why we used `ancient` here and other places? Might be no longer needed with this PR.